go.mod

```diff
@@ -13,13 +13,13 @@ require (
 	github.com/google/gops v0.3.18
 	github.com/gorilla/websocket v1.5.0
 	github.com/mitchellh/mapstructure v1.5.0
-	github.com/multiversx/mx-chain-core-go v1.2.0
+	github.com/multiversx/mx-chain-core-go v1.2.1-0.20230403113932-916b16d18978
```

Comment:
proper releases

Reply:
This will be merged in a feature branch. Will create a release tag when the feature branch will be merged.